docs/proposals/container-runtime-interface/capabilities.md

```diff
@@ -0,0 +1,137 @@
+        // unless the pod spec specifically drops the capability. You may not list a capability in both
+        // DefaultAddCapabilities and RequiredDropCapabilities.
+        // Note: this field is deprecated in favor of the DefaultCapabilities field.
+        DefaultAddCapabilities []api.Capability `json:"defaultAddCapabilities,omitempty"`
+        // RequiredDropCapabilities are the capabilities that will be dropped from the container.  These
+        // are required to be dropped and cannot be added.
+        // Note: this field is deprecated in favor of the DefaultCapabilities field.
+        RequiredDropCapabilities []api.Capability `json:"requiredDropCapabilities,omitempty"`
+        // AllowedCapabilities is a list of capabilities that can be requested to add to the container.
+        // Capabilities in this field may be added at the pod author's discretion.
+        // You must not list a capability in both AllowedCapabilities and RequiredDropCapabilities.
+        AllowedCapabilities []api.Capability `json:"allowedCapabilities,omitempty"`
+        // DefaultCapabilities is a list of capabilities that will always be applied to the container.
+        // In addition to the capabilities listed here, the pod author can add additional capabilities
+        // specified in the AllowedCapabilities field.
+        DefaultCapabilities []api.Capability `json:"defaultCapabilities,omitempty"`
+...
+}
+```
+
+The `Capabilities` struct in `pkg/api/types.go`, and `pkg/api/v1/types.go` gets a new `Default` field:
+
+```
+// Capabilities represent POSIX capabilities that can be added or removed to a running container to/from a default set.
+type Capabilities struct {
+        // Added capabilities to the default capabilities
+        Add []Capability `json:"add,omitempty"`
+        // Removed capabilities from the default capabilities
+        Drop []Capability `json:"drop,omitempty"`
+        // Default capabilities
+        Default []Capability `json:"default,omitempty"`
+}
+```
+
+With the changes above the runtime gets a complete capabilities execution context in the `PodSpec` with a default whitelist of capabilities, and a list of capabilities to be added/dropped.
+
+## Open questions:
+
+**Q:** What the default capability list should be?
+
+**A:** According to @smarterclayton, there has been questions and arguments about what the default capability list should be, and it's not settled down yet.
+       So for now, in order to keep backward compatibility, we'd better just define it as current [Docker's default capability list](https://docs.docker.com/engine/reference/run/#/runtime-privilege-and-linux-capabilities).
+
+**Q:** How wide does the default capability list apply? Does it apply to one node, or the whole cluster?
+
+**A:** A cluster-wide default capability list is proposed in this document. More specific per-node defaults can be discussed at a later point.
```

Comment:
I wouldn't consider this an open question. I think we can all agree that defaults should be cluster-wide for now.